content/kubermatic/master/guides/installation/add_seed_cluster_CE/_index.en.md

```diff
@@ -20,7 +20,7 @@ about the cluster relationships.
 Compared to master clusters, seed clusters are still mostly manually installed. Future versions of KKP
 will improve the setup experience further.
 
-When using Helm 2, install Tiller into the seed cluster first:
+If you're using Helm 2, install Tiller into the seed cluster first:
```

Comment:
I do not know if helm 2 is really an option here?

Reply:
Unfortunately it is